projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -271,6 +255,79 @@ export function asmB2bCustomerLists(): void {
   cy.get('cx-customer-list').should('not.exist');
 }
 
+export function asmB2bCustomerListPagination(): void {
```

Comment:
PaginationModule is a shared component, so the e2e tests don't check all the function details of it.